pupa/settings.py

```diff
@@ -72,3 +73,10 @@
 
 
 DATABASES = {'default': dj_database_url.parse(DATABASE_URL)}
+
+
+elasticsearch = None
```

Comment:
I'm not a super fan of having it in settings; we used to have `billy.core` or `pupa.core` for this before, but I think since we moved to Django, that's out. I wonder if it's worth re-adding a `core` module.